src/main/model/JWTHeader.java

```diff
@@ -0,0 +1,58 @@
+package main.model;
+//model for: JWT=$(hmcOktaJWT.py -o cnqr.okta.com -H 'HMC - US2' -e p00849947wds -R 22717526)
+public class JWTHeader {
+    private String PythonCall;
+    private String url;
```

Comment:
What is this URL for? Might want to be a little more specific - you'll likely have several URLs in this project once you're finished, you may want unique names, even if they don't conflict, so you know what you're working with. Keep in mind, documentation, whether in naming or in comments, is for you the dev.